src/auth.js

```diff
@@ -52,6 +52,14 @@ function * createUserWithEmailAndPassword (email, password) {
   return yield call([auth, auth.createUserWithEmailAndPassword], email, password)
 }
 
+function * sendEmailVerification (currentUser) {
```

Comment:
As with the other sagas I think we should be able to give this saga the same signature as Firebase's [`sendEmailVerification`](https://firebase.google.com/docs/reference/js/firebase.User#sendEmailVerification), i.e. it should take `actionCodeSettings` as parameter.

From the doc's example it seems that we can call this method directly from `auth.currenUser`:
```js
function * saga (actionCodeSettings) {
  const auth = this.app.auth()
  return yield call([auth.currentUser, auth.currentUser.sendEmailVerification], actionCodeSettings)
}
```

Reply:
And we'll need to implement [`applyActionCode`](https://firebase.google.com/docs/reference/js/firebase.auth.Auth#applyActionCode) as well to have the entire email verification flow.